website/docs/r/project_cluster.html.markdown

```diff
@@ -46,6 +46,8 @@ The following arguments are supported:
 
 * `enabled` - (Optional, boolean) Determines if cluster is active or not. Defaults to `true`.
 
+* `managed` - (Optional, boolean) Determines if cluster is managed by gitlab or not. Defaults to `true`.
```

Comment:
Please make a comment that we can not read that value in the doc

Reply:
will do. FYI: enabled works this way too, so I'll add it there too.